rapid/plugin/group_replication/tests/mtr/t/gr_single_primary_async_gr_start_on_boot.test

```diff
@@ -0,0 +1,230 @@
+--source include/assert.inc
+
+
+--let $rpl_connection_name= server2
+--source include/rpl_connection.inc
+
+--echo
+--echo # 5. Restart server2(secondary) with group replication start on boot
+--let $_group_replication_local_address= `SELECT @@GLOBAL.group_replication_local_address`
+--let $_group_replication_group_seeds= `SELECT @@GLOBAL.group_replication_group_seeds`
+--let $restart_parameters=restart:--group-replication-start-on-boot=1 --group-replication-group-name=$group_replication_group_name --group_replication_local_address=$_group_replication_local_address --group_replication_group_seeds=$_group_replication_group_seeds --group_replication_single_primary_mode=1 --group_replication_enforce_update_everywhere_checks=0
+--replace_result $group_replication_group_name GROUP_REPLICATION_GROUP_NAME $_group_replication_local_address GROUP_REPLICATION_LOCAL_ADDRESS $_group_replication_group_seeds GROUP_REPLICATION_GROUP_SEEDS
+--source include/restart_mysqld.inc
+
+--let $rpl_server_number= 2
+--source include/rpl_reconnect.inc
+
+--echo
+--echo # 6. Wait for GR to come ONLINE
+--let $group_replication_member_state= ONLINE
+--source ../inc/gr_wait_for_member_state.inc
+
+
+--echo
+--echo # 7. Create new channel ch3_2: server2(slave), server3(master)
+--replace_result $SERVER_MYPORT_3 SERVER_3_PORT
+--eval CHANGE MASTER TO MASTER_HOST='localhost', MASTER_USER='root', MASTER_PORT=$SERVER_MYPORT_3 for channel 'ch3_2'
+
+
+# variables for wait_for_slave_io_error.inc and wait_for_slave_sql_error.inc
+--let $rpl_channel_name=ch3_2
+
+--let $slave_io_errno= convert_error(ER_SLAVE_FATAL_ERROR)
+--let $show_slave_io_error= 0
+--let $slave_io_error_replace= / at [0-9]*/ at XXX/
+
+--let $slave_sql_errno= convert_error(ER_SLAVE_FATAL_ERROR)
+--let $show_slave_sql_error= 0
+--let $slave_sql_error_replace= / at [0-9]*/ at XXX/
+
+--echo
+--echo # 8. start slave and verify for error ER_SLAVE_FATAL_ERROR
+START SLAVE FOR CHANNEL 'ch3_2';
+--source include/wait_for_slave_io_error.inc
+--source include/wait_for_slave_sql_error.inc
+
+--echo
+--echo # 9. start slave io and verify for error ER_SLAVE_FATAL_ERROR
+START SLAVE IO_THREAD FOR CHANNEL 'ch3_2';
+--source include/wait_for_slave_io_error.inc
+
+--echo
+--echo # 10. start slave sql and verify for error ER_SLAVE_FATAL_ERROR
+START SLAVE SQL_THREAD FOR CHANNEL 'ch3_2';
+--source include/wait_for_slave_sql_error.inc
+
+--let $rpl_channel_name=
+
+--echo
+--echo # 11. Verify occurrence of error message
+--replace_regex /[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]T[0-9][0-9]:[0-9][0-9]:[0-9][0-9]\.[0-9][0-9][0-9][0-9][0-9][0-9][-+Z][0-9:]* *[0-9]* *\[/DATE_TIME [/
+
+--let $grep_file= $MYSQLTEST_VARDIR/tmp/gr_single_primary_async_gr_start_on_boot.err
+--let $grep_pattern= Can't start slave IO THREAD when group replication is running with single primary-mode on a secondary member
+--let $grep_output= print_count
```

Comment:
Probably broken with --repeat (grep_count will be 2x, 3x, ... for subsequent repeats)